build/linux/installer/conf/td-agent-bit-rs.conf

```diff
@@ -1,20 +1,28 @@
 [SERVICE]
-    Flush         30
+    HTTP_Server   Off
```

Comment:
This should technically be only a rename. Am i seeing this because you havent merged from ci_feature into this branch yet? or is this a weird merge issue?

Reply:
> This should technically be only a rename. Am i seeing this because you havent merged from ci_feature into this branch yet? or is this github caching issue?
> this file got changed recently in ci_feature branch https://github.com/microsoft/Docker-Provider/blob/ci_feature/installer/conf/td-agent-bit-rs.conf after I created ci_dev branch. This is expected difference. just verified all the files between ci_feature and this branch and everything looks fine no differences.